osbrain/tests/test_agent_pubsub_topics.py

```diff
@@ -143,27 +144,65 @@ def test_pubsub_topics_raw(nsproxy, serializer):
     time.sleep(0.1)
 
     # Check each agent received the corresponding messages
-    assert message_01 in a1.get_attr('received_list')
-    assert b'fooWorld' in a1.get_attr('received_list')
-    assert b'foobarFOO' in a1.get_attr('received_list')
-    assert b'foBAR' in a1.get_attr('received_list')
-
-    assert message_01 not in a2.get_attr('received_list')
-    assert b'fooWorld' in a2.get_attr('received_list')
-    assert b'foobarFOO' in a2.get_attr('received_list')
-    assert b'foBAR' not in a2.get_attr('received_list')
-
-    assert message_01 not in a3.get_attr('received_list')
-    assert b'fooWorld' in a3.get_attr('received_list')
-    assert b'foobarFOO' in a3.get_attr('received_list')
-    assert b'foBAR' not in a3.get_attr('received_list')
-
-    assert message_01 not in a4.get_attr('received_list')
-    assert b'fooWorld' not in a4.get_attr('received_list')
-    assert b'foobarFOO' not in a4.get_attr('received_list')
-    assert b'foBAR' not in a4.get_attr('received_list')
-
-    assert message_01 not in a5.get_attr('received_list')
-    assert b'fooWorld' in a5.get_attr('received_list')
-    assert b'foobarFOO' in a5.get_attr('received_list')
-    assert b'foBAR' in a5.get_attr('received_list')
+    assert message_01 in a1.get_attr('received')
+    assert b'fooWorld' in a1.get_attr('received')
+    assert b'foobarFOO' in a1.get_attr('received')
+    assert b'foBAR' in a1.get_attr('received')
+
+    assert message_01 not in a2.get_attr('received')
+    assert b'fooWorld' in a2.get_attr('received')
+    assert b'foobarFOO' in a2.get_attr('received')
+    assert b'foBAR' not in a2.get_attr('received')
+
+    assert message_01 not in a3.get_attr('received')
+    assert b'fooWorld' in a3.get_attr('received')
+    assert b'foobarFOO' in a3.get_attr('received')
+    assert b'foBAR' not in a3.get_attr('received')
+
+    assert message_01 not in a4.get_attr('received')
+    assert b'fooWorld' not in a4.get_attr('received')
+    assert b'foobarFOO' not in a4.get_attr('received')
+    assert b'foBAR' not in a4.get_attr('received')
+
+    assert message_01 not in a5.get_attr('received')
+    assert b'fooWorld' in a5.get_attr('received')
+    assert b'foobarFOO' in a5.get_attr('received')
+    assert b'foBAR' in a5.get_attr('received')
```

Comment:
All changes related to receive, received_list, etc. should go in a separate pull-request.